web/modules/custom/joinup_subscription/joinup_subscription.post_update.php

```diff
@@ -29,6 +29,11 @@ function joinup_subscription_post_update_subscribe_to_community_content(&$sandbo
     ->execute();
 
   foreach (OgMembership::loadMultiple($mids) as $membership) {
+    // Avoid breaking the update path for leftover memberships.
```

Comment:
This comment makes it seem that this is a workaround, but it is not. Let's instead explain why we really do this, maybe something like this:

```
    // Organic Groups can be configured to delete memberships asynchronously in 
    // a cron job or a manually started batch process. This means orphaned
    // memberships can be present in the database. Ignore them.
```